customize/packer.lua

```diff
@@ -423,6 +423,14 @@ fi
 %s
 %s
 
+# set ownership to kong user/group
+find /usr/local/share/lua/5.1/          -type f -exec chown 1000:1000 "{}" +
```

Comment:
@gszr maybe this can be changed to `chown kong:kong`. wdyt?

Reply:
what about `s/1000/id -u/g`

Reply:
@hutchic sorry, didn't get that?

Reply:
instead of assuming `1000:1000` can we get the uid:gid with the `id` command?

Reply:
so I tried this:
```
KUSR=$(id -u kong)
KGRP=$(id -g kong)
find /usr/local/share/lua/5.1/          -type f -exec chown $KUSR:$KGRP "{}" +
find /usr/local/share/lua/5.1/          -type d -exec chown $KUSR:$KGRP "{}" +
```

But the result is
```
/usr/local/share/lua/5.1 $ ls -l
total 344
-rw-r--r--    1 kong     nogroup      29426 Dec 15 18:19 MessagePack.lua
-rw-r--r--    1 kong     nogroup       2788 Dec 15 18:19 ansicolors.lua
-rw-r--r--    1 kong     nogroup      12069 Dec 15 18:19 binaryheap.lua
drwxr-xr-x    1 kong     nogroup       4096 Dec 17 19:24 cassandra
-rw-r--r--    1 kong     nogroup      32158 Dec 15 18:19 date.lua
-rw-r--r--    1 kong     nogroup      10591 Dec 15 18:19 etlua.lua
-rw-r--r--    1 kong     nogroup      10167 Dec 15 18:19 ffi-zlib.lua
-rw-r--r--    1 kong     nogroup       9543 Dec 15 18:19 inspect.lua
```

Which I find weird, originally it was;
```
/usr/local/share/lua/5.1 $ ls -l
total 296
-rw-r--r--    1 1000     1000         29426 Dec 15 18:19 MessagePack.lua
-rw-r--r--    1 1000     1000          2788 Dec 15 18:19 ansicolors.lua
-rw-r--r--    1 1000     1000         12069 Dec 15 18:19 binaryheap.lua
drwxr-xr-x    2 1000     1000          4096 Dec 17 19:24 cassandra
-rw-r--r--    1 1000     1000         32158 Dec 15 18:19 date.lua
-rw-r--r--    1 1000     1000         10591 Dec 15 18:19 etlua.lua
-rw-r--r--    1 1000     1000         10167 Dec 15 18:19 ffi-zlib.lua
-rw-r--r--    1 1000     1000          9543 Dec 15 18:19 inspect.lua
```

Using the `id` command on alpine gives me this:
```
/usr/local/share/lua/5.1 $ id -u kong
100
/usr/local/share/lua/5.1 $ id -g kong
65533
```

Not sure where we do it, but seems we set the ownership to `1000:1000`, and not to `kong:kong` in the original docker builds.

So how should this customize behave?